supersim_test.go

```diff
@@ -65,6 +65,15 @@ type TestSuite struct {
 	Supersim *Supersim
 }
 
+type JSONL2ToL2Message struct {
+	Destination uint64         `json:"Destination"`
+	Source      uint64         `json:"Source"`
+	Nonce       *big.Int       `json:"Nonce"`
+	Sender      common.Address `json:"Sender"`
+	Target      common.Address `json:"Target"`
+	Message     hexutil.Bytes  `json:"Message"`
+}
```

Comment:
you can prob just import this from admin.JSONL2ToL2Message

In a future PR might make sense to unify with the indexer's message type so we don't have two types. As discussed, out of scope for this PR! just noting.

Reply:
noted